src/App.tsx

```diff
@@ -15,48 +16,109 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
```

Comment:
JFYI, usually, we enum constants names are just capitalized

```suggestion
enum SortType {
  None,
  Alphabet,
  Length,
}
```